aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandlerTest.java

```diff
@@ -54,16 +58,25 @@ public void setup() {
     }
 
     @Test
-    public void testSuccessState() {
-        doReturn(CreateProfilingGroupResponse.builder()
+    public void testSuccessStateNoPermissions() {
```

Comment:
It maybe a good time to change the testing structure and start adapting Nested here.

See https://www.petrikainulainen.net/programming/testing/junit-5-tutorial-writing-nested-tests/.

Reply:
Good suggestion, I will update them to use Junit 5's nested feature, thanks.